Marlin/pinsDebug.h

```diff
@@ -724,75 +178,130 @@ static bool pwm_status(uint8_t pin) {
   SERIAL_PROTOCOLPGM("  ");
 } // pwm_status
 
+
+#define COM_UNSHIFTED(T,L) (TCCR##T##A & (_BV(COM##T##L##1) | _BV(COM##T##L##0)))
+
+#define COM_PRINT(N,Z)  do { \
+     char temp_string[] = STRINGIFY(Z); \
+     switch(temp_string[0]) { \
+      case 'A' : \
+        SERIAL_PROTOCOLPAIR("    COM" STRINGIFY(N) "" STRINGIFY(Z) ": ", (COM_UNSHIFTED(N,Z) >> 6)); \
+        break; \
+      case 'B' : \
+        SERIAL_PROTOCOLPAIR("    COM" STRINGIFY(N) "" STRINGIFY(Z) ": ", (COM_UNSHIFTED(N,Z) >> 4)); \
+        break; \
+      case 'C' : \
+        SERIAL_PROTOCOLPAIR("    COM" STRINGIFY(N) "" STRINGIFY(Z) ": ", (COM_UNSHIFTED(N,Z) >> 2)); \
+        break; \
+      default:  \
+        SERIAL_PROTOCOLPAIR("  com_make L:",  temp_string); \
+        SERIAL_PROTOCOLPGM("spacer");\
+        for (uint8_t y = 0; y < 28; y++) {     \
+          if (temp_string[y] != 0) MYSERIAL.write(temp_string[y]);\
+          else break;\
+        }\
+        break; \
+    } \
+  } while(0);
+
+
+
+#define ICR_VALUE(N)  (ICR##N)
+#define CS_MAKE(N) (TCCR##N##B & (_BV(CS##N##0) | _BV(CS##N##1) | _BV(CS##N##2)) )
 #define WGM_MAKE3(N) (((TCCR##N##B & _BV(WGM##N##2)) >> 1) | (TCCR##N##A & (_BV(WGM##N##0) | _BV(WGM##N##1))))
 #define WGM_MAKE4(N) (WGM_MAKE3(N) | (TCCR##N##B & _BV(WGM##N##3)) >> 1)
 #define TIMER_PREFIX(T,L,N) do{ \
```

Comment:
`TIMER_PREFIX` is one of the macros that can be made more economical by calling a function, passing only the parameters that differ. The reason this would shrink the binary is that the compiler doesn't combine identical PROGMEM strings into a single string in the DATA block, as it does with heap strings.